.github/workflows/pr_event_moves_card.yml

```diff
@@ -0,0 +1,123 @@
+              #        --url https://api.github.com/graphql \
+              #        --header "Authorization: Bearer ${{ secrets.PAT }}" \
+              #        --data "{\"query\":\"{ node(id: \\\"$project_id\\\") { ... on ProjectV2 { items(first: 100) { nodes { id fieldValues(first: 8) { nodes { ... on ProjectV2ItemFieldTextValue { text field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldDateValue { date field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldSingleSelectValue { name field { ... on ProjectV2FieldCommon { name } } } } } content { ... on Issue { id number repository { id } } } } } } } }\"}")
+              #echo "issues=$issues"
+              project_fields_and_issues=$(curl --request POST \
+                --url https://api.github.com/graphql \
+                --header "Authorization: Bearer ${{ secrets.PAT }}" \
+                --data "{\"query\":\"{ node(id: \\\"$project_id\\\") { ... on ProjectV2 { items(first: 100) { nodes { id fieldValues(first: 8) { nodes { ... on ProjectV2ItemFieldTextValue { text field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldDateValue { date field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldSingleSelectValue { name field { ... on ProjectV2FieldCommon { name } } } } } content { ... on Issue { id number repository { id } } } } } fields(first: 20) { nodes { ... on ProjectV2SingleSelectField { id name options { id name } } } } } } }\"}")
+              echo "issues=$project_fields_and_issues"
+              # Extract the issue ids, PVII ids, and issue numbers. The issue_id is the global_id, the PVII id is the project specific id, and the issue number is the issue number (eg. #123)
+              # The issue number is not unique over all projects. So we check if the repository of the issue and the PR are the same.
+              issue_numbers=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].content.number')
+              
+              issue_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].content.id')
+              PVII_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].id')
+              # extract the repository id from the issue
+              repository_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].content.repository.id')
+              # In the following loop we need all three values, so we combine them into a single string
+              combined_ids=$(paste -d, <(echo "$issue_ids") <(echo "$PVII_ids") <(echo "$issue_numbers") <(echo "$repository_ids"))
+              for combined_id in $combined_ids; do
+                issue_id=$(echo $combined_id | cut -d, -f1)
+                PVII_id=$(echo $combined_id | cut -d, -f2)
+                issue_number=$(echo $combined_id | cut -d, -f3)
+                repository_id=$(echo $combined_id | cut -d, -f4)
+                echo "issue_id is $issue_id"
```

Comment:
do we need these outputs? Could be a very long list. Maybe only if the issue number is matching?

Reply:
No, this issue_id outputs the issue id of all issues linked to the project. There can be many issues and maybe we have many projects one time. I think the output in line 59 is enough.

Reply:
Ah, now I understand. It was helpfull debug-output, but now we probably don't need it anymore. I will delete it.